src/core/dbus-execute.c

```diff
@@ -3338,25 +3338,23 @@ int bus_exec_context_set_transient_property(
                 if (r < 0)
                         return r;
 
-                if (!isempty(s)) {
```

Comment:
What was the intent of originally having it be empty? I am trying to work out how this was intended to work/what it was intended to mean previously.

Reply:
The linked commit changed it without justification, and it's wrong. Hence I don't think that matters much.

Reply:
Yeah, that's my bad...